src/xo-app/new-vm/index.js

```diff
@@ -1220,6 +1221,15 @@ export default class NewVm extends BaseComponent {
             &nbsp;
             {_('newVmBootAfterCreate')}
           </Item>
+          <Item>
+            <input
+              checked={autoPoweron}
+              onChange={this._getOnChangeCheckbox('autoPoweron')}
+              type='checkbox'
+            />
```

Comment:
To match the `bootAfterCreate` checkbox right next to it. Should I change both of them?